### PR TITLE
[232347] Ensure that future time is returned from a mocked function

### PR DESCRIPTION
**Pull request by irisTa56**
https://acsmine.tok.access-company.com/redmine/issues/232347#note-23

### CODE REVIEW

**Review by Fumipo-Theta — APPROVED** [review]
LGTM!

**Review by aMasakiTakahashi — APPROVED** [review]
LGTM!

File: test/async_job_test.exs [hunk 1]
```diff
@@ -236,14 +236,15 @@ defmodule Testgear.AsyncJobTest do
   end
 
   test "recurring job should be requeued on completion" do
+    :meck.new(Cron, [:passthrough])
```
Comment (aYukiSekiguchi) [inline test/async_job_test.exs:239]: It would be better if you write a comment about why `new/2` must be placed here to avoid regression.

**Review by aYukiSekiguchi — APPROVED** [review]
LGTM

File: test/async_job_test.exs (hunk 1 above)
Comment (irisTa56) [reply test/async_job_test.exs:241]: Thank you.
I add a comment at https://github.com/access-company/testgear/pull/22/commits/c7cc49a7b286b09f7490f1f45a532f9b4f221061

**Pull request merged**